pkg/jobqueue/dbjobqueue/dbjobqueue.go

```diff
@@ -234,8 +251,8 @@ func (q *DBJobQueue) Enqueue(jobType string, args interface{}, dependencies []uu
 	}
 	defer func() {
 		err := tx.Rollback(context.Background())
-		if err != nil && !errors.Is(err, pgx.ErrTxClosed) {
-			logrus.Error("error rolling back enqueue transaction: ", err)
+		if err != nil && !errors.As(err, &pgx.ErrTxClosed) {
```

Comment:
Bad merge (breaks build by Go 1.19).

Reply:
Damn, I have many conflicts and I hoped to catch them all (TM).

Good eyes, rebased!